doc/references/ImplementationOfVHDL.rst

```diff
@@ -142,21 +136,77 @@ You can either use a default clock like this:
 
 .. code-block:: VHDL
 
-    -- psl default clock is rising_edge (CLK);
-    -- psl assert always
-    --   a -> eventually! b;
+    default clock is rising_edge (CLK);
+    assert always
+      a -> eventually! b;
 
 or use a clocked expression (note the use of parentheses):
 
 .. code-block:: VHDL
 
-    -- psl assert (always a -> next[3](b)) @rising_edge (clk);
+    assert (always a -> next[3](b)) @rising_edge(clk);
 
 
 Of course only the simple subset of PSL is allowed.
 
 Currently the built-in functions are not implemented.
 
+PSL usage
+-----------------
+
+PSL annotations embedded in comments
```

Comment:
Instead of indenting the content that belongs to this (sub)section, I'd use `^` to underline the heading.

https://documentation-style-guide-sphinx.readthedocs.io/en/latest/style-guide.html#headings